var/spack/repos/builtin/packages/linaro-forge/package.py

```diff
@@ -23,6 +23,9 @@ class LinaroForge(Package):
     maintainers("kenche-linaro")
 
     if platform.machine() in ["aarch64", "arm64"]:
```

Comment:
Verified new hash for aarch64 and x86_64.

Unable to download from url https://downloads.linaroforge.com/24.0.4/linaro-forge-24.0.4-linux-arm64.tar for `platform.machine() == "arm64"`.  I don't think `platform.machine()` can have that value, though. `spack arch` doesn't list it.

Reply:
Modified the pull request to remove `arm64` and updated commit message.